src/js/components/Video/Video.js

```diff
@@ -48,7 +48,7 @@ const Video = forwardRef(
       alignSelf,
       autoPlay,
       children,
-      controls = 'over',
```

Comment:
Why can't we preserve the existing default here? The caller can pass 'over', so we have to be prepared to handle it anyway.